src/cmd/webhook/builder.go

```diff
@@ -97,7 +97,12 @@ func (builder CommandBuilder) buildRun() func(*cobra.Command, []string) error {
 			return err
 		}
 
-		if !builder.isDeployedViaOlm {
+		isDeployedViaOLM, err := kubesystem.IsDeployedViaOlm(webhookManager.GetClient(), builder.podName, builder.namespace)
```

Comment:
IsDeployedViaOlm could also be wrapped in a function like in the operator builder